activiti-cloud-dependencies/pom.xml

```diff
@@ -51,67 +40,67 @@
     <dependency>
       <groupId>org.activiti.cloud</groupId>
       <artifactId>activiti-cloud-build-parent</artifactId>
-      <version>${activiti-cloud-build.version}</version>
+      <version>${activiti-cloud.version}</version>
       <type>pom</type>
     </dependency>
     <dependency>
       <groupId>org.activiti.cloud</groupId>
       <artifactId>activiti-cloud-api-dependencies</artifactId>
-      <version>${activiti-cloud-api.version}</version>
+      <version>${activiti-cloud.version}</version>
       <type>pom</type>
     </dependency>
     <dependency>
       <groupId>org.activiti.cloud</groupId>
       <artifactId>activiti-cloud-service-common-dependencies</artifactId>
-      <version>${activiti-cloud-service-common.version}</version>
+      <version>${activiti-cloud.version}</version>
       <type>pom</type>
     </dependency>
     <dependency>
       <groupId>org.activiti.cloud</groupId>
       <artifactId>activiti-cloud-acceptance-tests-dependencies</artifactId>
-      <version>${activiti-cloud-acceptance-tests.version}</version>
+      <version>${project.version}</version>
```

Comment:
This is referencing should be `${activiti-cloud.version}` as it's referencing https://github.com/Activiti/activiti-cloud/blob/develop/activiti-cloud-acceptance-tests/activiti-cloud-acceptance-tests-dependencies/pom.xml#L10

Reply:
I should have marked as WIP :)